frontend/model/contracts/identity.js

```diff
@@ -40,6 +43,14 @@ DefineContract({
           Vue.delete(state.attributes, attribute)
         }
       }
+    },
+    'gi.contracts/identity/updateSettings': {
+      validate: object,
+      process ({ data }, { state }) {
+        for (var key in data) {
+          Vue.set(state.settings, key, data[key])
+        }
+      }
```

Comment:
This is the user's public profile. I don't think it should be used to set settings... Plus, we already have an `attributes` key that is implemented identically and can be used for that purpose. So I would remove this to DRY.

Reply:
Oh, I thought `attributes` would be like a "public profile" where anyone can have access. That's why I created `settings`, more like a private part with settings - appLogsfilters, notifications (newsletters), appearance, etc... Don't you think it makes sense to have two parts?

Reply:
Ah, I see. It actually does make sense. Can you add that explanation here in a comment then? (But still save these `appLogsFilter` settings in vuex)

Reply:
So, just to clarify. We'll keep the `updateSettings` in this PR because we know it will be used in a later PR, but the `appLogsFilter` is only saved on vuex state and not on the user contract.

Reply:
Yep! 😄 👍

Reply:
Done, please check the new code